pkg/deviceshifu/deviceshifulwm2m/deviceshifulwm2mconfig_test.go

```diff
@@ -0,0 +1,84 @@
+package deviceshifulwm2m
+
+import (
+	"testing"
+
+	"github.com/edgenesis/shifu/pkg/deviceshifu/deviceshifubase"
+)
+
+func TestCreateLwM2MInstructions(t *testing.T) {
+	tests := []struct {
+		InstructionName string
+		input           *deviceshifubase.DeviceShifuInstructions
+		expected        *LwM2MInstruction
+		expectingPanic  bool
```

Comment:
when will except panic?

Reply:
judge whether the returned result is panic